tests/unittests/config/test_cc_set_passwords.py

```diff
@@ -586,15 +586,21 @@ def _get_str_class_num(self, str):
             (20, 4),
         ],
     )
-    def test_rand_user_password(self, strlen, expected_result):
+    @pytest.mark.parametrize("type", ["RANDOM", "R", "RANDOM_FRIENDLY"])
+    def test_rand_user_password(self, strlen, expected_result, type):
         if expected_result is ValueError:
             with pytest.raises(
                 expected_result,
                 match="Password length must be at least 4 characters.",
             ):
                 setpass.rand_user_password(strlen)
         else:
-            rand_password = setpass.rand_user_password(strlen)
+            rand_password = setpass.rand_user_password(strlen, type=type)
```

Comment:
Thx for the unittest, can we also manipulate the existing tests/integration-tests/modules/test_set_password.py to make sure we also cover both `R` `RANDOM` `RANDOM_FRIENDLY` or `RANDOM_UNAMBIGUOUS`.

Reply:
note defaults of tox -e integraiton-tests -- tests/integration-tests/modules/test_set_password.py should work as long as you have:
```bash
cat > .config/pycloudlib.toml <<EOF
[lxd]
EOF
```